src/Compilers/CSharp/Portable/Binder/SwitchBinder.cs

```diff
@@ -267,73 +253,62 @@ private BoundExpression ConvertCaseExpression(TypeSymbol switchGoverningType, CS
                     diagnostics.Add(ErrorCode.WRN_GotoCaseShouldConvert, node.Location, switchGoverningType);
```

Comment:
The handling of goto case is identical in both language versions.

---
In reply to: [70300809](https://github.com/dotnet/roslyn/pull/12416#discussion_r70300809) [](ancestors = 70300809)